### DISCUSSION TITLE
Not able to start Next app

### DISCUSSION THREAD

**Discussion by Jassi10000-zz** [vercel/next.js#30462] [Help]
### What version of Next.js are you using?

latest

### What version of Node.js are you using?

v14.15.4

### What browser are you using?

Chrome

### What operating system are you using?

Windows

### How are you deploying your application?

next start

### Describe the Bug

I have built a next app with tailwind css template but it doesn't seems to run, even on getting about the error I tried to resolve it but same result

![image](https://user-images.githubusercontent.com/64856348/137969431-f1ace9fc-1164-437e-aede-3ffade3d21ee.png)

### Expected Behavior

The next app should be able to run successfully on localhost

### To Reproduce

npm run dev

**Comment by ijjk** [comment]
Hi, the provided reproduction steps do not reproduce the error, please provide full steps taken. Was `npm install` run fully before running `npm run dev`? Does clearing `node_modules` and re-installing fix it?

**Comment by Jassi10000-zz** [comment]
> Hi, the provided reproduction steps do not reproduce the error, please provide full steps taken. Was `npm install` run fully before running `npm run dev`? Does clearing `node_modules` and re-installing fix it?

Yes I have done <code> npm install </code> and then did <code> npm run dev </code> 
Even when I got the error that next is not found  , I tried installing via <code> npm install next </code> but results were same

Talking about clearing and restalling <code> node_modules </code> doesn't fix the issue . Already tried that 2-3 times

**Comment by ijjk** [comment]
Can you share the `package.json` or repo for the project you are seeing this on?

**Comment by Jassi10000-zz** [comment]
sure , am sharing the <code>package.json</code>

```
{
  "private": true,
  "scripts": {
    "dev": "next dev",
    "build": "next build",
    "start": "next start"
  },
  "dependencies": {
    "next": "latest",
    "react": "^17.0.2",
    "react-dom": "^17.0.2"
  },
  "devDependencies": {
    "autoprefixer": "^10.2.6",
    "postcss": "^8.3.5",
    "tailwindcss": "^2.2.4"
  }
}

```

**Comment by ijjk** [comment]
I'm still not able to reproduce with the provided info, please provide a repo with a minimal reproduction if you're still encountering this.

**Reply by Jassi10000-zz** [reply]
Sure , will put the link in a while
